### ISSUE TITLE
aichat: claude should output formatted code

### ISSUE THREAD

**Issue by nullhook** [brave/brave-browser#35380]
only llama outputs formatted code but claude doesn't.

**Comment by kjozwiak** [comment]
Adding `QA/Blocked` for now as the above was merged into `cr121-1.62.x` via https://github.com/brave/brave-core/pull/21593. Once `C121` is merged into `1.62.x` via https://github.com/brave/brave-core/pull/21593, we'll unblock the above and mention the needed `1.62.x` version for verification 👍

**Comment by kjozwiak** [comment]
The above requires `1.62.148` or higher for `1.62.x` verification 👍

**Comment by stephendonner** [comment]
## Verified `IN-PROGRESS` using

```
Brave | 1.62.149 Chromium: 121.0.6167.75 (Official Build) (x86_64)
-- | --
Revision | ff84587bd70af9fcbcbe59fc5194ca65082759c4
OS | macOS Version 14.3 (Build 23D56)
```

**Comment by Uni-verse** [comment]
In Progress on `Samsung Galaxy S21` using the following version: 

```
Brave	1.62.152 Chromium: 121.0.6167.101 (Official Build) (64-bit) 
Revision	43e7bbb904a2fa762d73f3ac732effecba4c5362
OS	Android 13; Build/TP1A.220624.014; 33; REL
```
